client/src/components/InputsModal/InputsModal.tsx

```diff
@@ -211,13 +163,8 @@ const InputsModal: FC<InputsModalProps> = ({
   };
 
   const handleSubmitKeydown = (e: React.KeyboardEvent<HTMLDivElement>) => {
-    if (
-      open &&
-      e.key === 'Enter' &&
-      (e.metaKey || e.ctrlKey) &&
-      !missingRequiredInput &&
-      !invalidInput
-    ) {
+    const opKey = window.navigator.userAgent.includes('Mac') ? e.metaKey : e.ctrlKey;
```

Comment:
Tested on Mac and this works as expected - cmd+enter submits, ctrl+enter doesn't. I don't have a windows machine to easily test with at the moment but I could try tonight if we want to confirm that. 
Personally I wouldn't object to both key combinations working on both OSes, partly because the user agent testing on this line feels like there's a chance of a false positive (maybe the word Machine or something? just making things up.)
But also tbh it wasn't apparent to me in the UI that there is a special combination to submit the form in the first place, so I'm not sure what would be "least surprising".
Either way, I'm approving because it does what it says it does and meets the def of done from the ticket.

Reply:
Yeah this feature is mainly just a developer shortcut so shouldn't really be known anyways. I limited the keybindings because the ctrl key was behaving differently than the cmd key on MacOS for whatever reason but since I resolved the original issue this shouldn't come up again. So at this point it would be a matter of preference whether we allow both; I doubt there would be technical issues on the userAgent but IMO having both is probably easier on devs in general. The one thing to note is there is a slight delay with using ctrl vs cmd.